loopfib.c

```diff
@@ -0,0 +1,15 @@
+#include<stdio.h>
```

Comment:
Close, but you don't handle edge cases well! So when you put 0 or 1 fibonacci numbers, it outputs the first two numbers regardless. Also, add comments!